scripts/seed.ts

```diff
@@ -30,6 +30,10 @@ async function main() {
 		);
 	}
 
+	if (org && org.name == 'Cardboard') {
+		console.log('database may already be seeded, may not create new data.');
```

Comment:
this should return early

Reply:
drop every other change in this file except for this and make it return early.

Reply:
Not sure what you mean by drop every other change, returns early in 9eb0f50bab64338f7af5718af500c50620f4d355